### PR TITLE
[release-3.9] [Bug 1614916] Default 3.9 to use RPM based CRI-O install

### PR DESCRIPTION
**Pull request by mtnbikenc**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1614916

### CODE REVIEW

**Comment by mtnbikenc** [comment]
The gcp test is still failing because the 3.9 origin image is still out of date.

File: inventory/hosts.example [hunk 1]
```diff
@@ -147,10 +147,8 @@ debug_level=2
 # The following options must not be used
 # - openshift_docker_options
 #openshift_docker_use_system_container=False
-# Install and run cri-o. By default this will install cri-o as a system container.
+# Install and run cri-o.
 #openshift_use_crio=False
-# You can install cri-o as an rpm by setting the following variable:
-#openshift_crio_use_rpm=False
```
Comment (vrutkovs) [inline inventory/hosts.example]: Should we add `openshift_crio_use_rpm` to `__warn_deprecated_vars`?

Comment (michaelgugino) [reply inventory/hosts.example]: Yeah, we should do that.  Make a note to just remove in 3.12, no need to fail if that's defined as it won't really affect anything.

**Comment by mtnbikenc** [comment]
@vrutkovs @michaelgugino Updated.

**Comment by vrutkovs** [comment]
GCP test won't pass, merged manually

**Pull request merged**